pkg/data/components/HoneycombExporter.yaml

```diff
@@ -51,6 +60,52 @@ properties:
       - oneof(all, none)
     default: all
     advanced: true
+  - name: Headers
```

Comment:
I don't think we should be applying headers here.

The only settings should be the ingest key and the metrics dataset. The other things could potentially be advanced settings but they only apply to one of the exporters